Src/PerformanceCollector/Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -191,12 +191,10 @@ private static ITelemetryDocument ConvertRequestToTelemetryDocument(RequestTelem
                            Timestamp = requestTelemetry.Timestamp,
                            OperationId = TruncateValue(requestTelemetry.Context?.Operation?.Id),
                            Name = TruncateValue(requestTelemetry.Name),
-                           StartTime = requestTelemetry.StartTime,
                            Success = IsRequestSuccessful(requestTelemetry),
                            Duration = requestTelemetry.Duration,
                            ResponseCode = requestTelemetry.ResponseCode,
                            Url = requestTelemetry.Url,
```

Comment:
Are you guys removing StartTime and HttpMethod from RequestTelemetry? Are there alternatives?